(DOCSP-23102) Log Connection Settings (#230)
mongodb/docs-java: source/fundamentals/connection/mongoclientsettings.txt

   expects to connect to an available socket within ``5 SECONDS`` before
   timing out.

.. tip:: Log Your Settings

>    To log the settings you apply to your ``MongoClient`` instance,
>    ensure you log ``INFO`` level messages from the
>    ``org.mongodb.driver.client`` logger.

Reviewer: ```suggestion
   To log the ``MongoClient`` instance settings,
   set the ``org.mongodb.driver.client`` named
    logger to the ``INFO`` level.
```
